### DISCUSSION TITLE
CoPilot acces

### DISCUSSION THREAD

**Discussion by abhisjoshi** [community/community#10520] [Copilot Conversations]
Hi Team, What is the process of getting access to the copilot solution. I am trying to test this for some algorithms and usecases, is it possible to get the access to it? Looking forward to your hearing from you.

**Comment by cpholguera** [comment]
Hi @abhisjoshi, this us the wrong category. Your question is about "GitHub Copilot". You should change the category from "GitHub Discussions" to "GitHub Copilot" 🙂
